cmd/mungedocs/analytics.go

```diff
@@ -37,6 +38,10 @@ var (
 
 // This adds the analytics link to every .md file.
 func checkAnalytics(fileName string, fileBytes []byte) (output []byte, err error) {
+	if strings.HasSuffix(fileName, "types.go") {
```

Comment:
We definitely can't make all the mungers filter for the type of file like this.

If you want to do this in the current munger, then you need to add a string list to the munger list in mungedocs.go, so each munger declares what file types it is willing to run on.